### ISSUE TITLE
About Training Process & Result.

### ISSUE THREAD

**Issue by Jaccelerator** [FengZhu-Joey/GA-DTCDR#11]
I ran the dataset `douban_book_music`, encoutering the problem that the nan loss and the test metric is fixed. Would you kindly tell how to solve it, e.g., tuning what the parameters.

<details>
<summary>Experimental Results Log.</summary>

Start Evaluation!
Epoch  279 Domain A: douban_book TopK: 10 HR: 0.34828009828009826, NDCG: 0.19299960999304305
Epoch  279 Domain B: douban_music TopK: 10 HR: 0.2963302752293578, NDCG: 0.16454914953441419
====================Epoch  279 End====================
====================Epoch  280 ==================== 
170 / 178 : loss = nan;1076660156255;
Mean loss in this epoch is: Domain A=nan;Domain B=nan
==================================================  
Start Evaluation!
Epoch  280 Domain A: douban_book TopK: 10 HR: 1.0, NDCG: 0.33333333333333337
Epoch  280 Domain B: douban_music TopK: 10 HR: 1.0, NDCG: 0.3333333333333334
</details>
